lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -198,7 +198,7 @@ div.action span .nextbutton, .Mrphs-siteHierarchy span .nextbutton{
 
 }
 .act .deleteButton {
-    float: none
+    float: right
```

Comment:
Is this going to be too far to the right on wide screens and easily missed by users looking for it?